dist/redhat/scylla.spec

```diff
@@ -103,15 +97,14 @@ fi
 /usr/bin/systemctl daemon-reload ||:
 
 %posttrans server
-if  [ -d /tmp/%{name}-%{version}-%{release} ]; then
-    cp -a /tmp/%{name}-%{version}-%{release}/* /etc/scylla/
-    rm -rf /tmp/%{name}-%{version}-%{release}/
+if [ -d /tmp/scylla-%{version}-%{release} ]; then
```

Comment:
unrelated to this patch, but why are we using /tmp directly, as is? mktemp and friends... ?